### ISSUE TITLE
'end turn' button during formation phase

### ISSUE THREAD

**Issue by vaendryl** [vaendryl/Sunrider#137]
> Well, I have a least pressing thing to mark. Sometimes, during deployment, instead of the grey "Start" button, the red button for "End Turn" appears, and clicking it makes the game crash. It has happened to me, actually, when I loaded from the main menu a game saved during deployment.
Reloading and using auto-saves helps, though, so it is definitely not urgent.

**Comment by DoumanAsh** [comment]
It happens only when you hover mouse over some unit

**Comment by vaendryl** [comment]
it's certainly caused by BM.phase as this directly instructs the battle_screen which button to show. 

loading a save made in formation mode loads it with BM.phase still in 'formation', as shown by the button. the moment you pick up a unit though it somehow changes to 'Player'. reloading the same save  somehow prevents this from happening.

in fact, just opening en closing the dev console fixes this!!

I have _no_ idea how this is possible.

**Comment by DoumanAsh** [comment]
Yes... i could not follow the whole way so i'm unsure how to debug it.

Btw BM is stored in RenPy's store and save file should contain all data of its store?

**Comment by vaendryl** [comment]
I honestly have very little clue how saving and loading truly works. I know it's complicated and tightly woven into the basic fabric of how ren'py works in general.
